tests/terraform_setup/ec2/terraform.tf

```diff
@@ -28,7 +25,7 @@ locals {
   aws_ecs_service_stack_name = "${var.aws_resource_prefix}-svc-stack"
 
   # The name of the ECR repository to be created
-  aws_ecr_repository_name = var.aws_resource_prefix
+  aws_ecr_repository_name = "${var.aws_resource_prefix}"
```

Comment:
In theory there is no need to put this variable inside a string.

Reply:
As the other, there's no need, it's there for easier name changing if required.